lib/main_common.pm

```diff
@@ -664,14 +664,10 @@ sub load_bootloader_s390x {
 sub boot_hdd_image {
     # On JeOS we don't need to load any test to boot, but to keep main.pm sane just return.
     is_jeos() ? return 1 : get_required_var('BOOT_HDD_IMAGE');
-    if (check_var("BACKEND", "svirt")) {
-        loadtest "installation/bootloader_svirt" unless load_bootloader_s390x();
-    }
-    if (get_var('UEFI') && (get_var('BOOTFROM') || get_var('BOOT_HDD_IMAGE'))) {
-        loadtest "boot/uefi_bootmenu";
```

Comment:
Why was condition and `loadtest "boot/uefi_bootmenu"` removed? Is it  because of bsc#1022064 ?

Reply:
This part is weird (or old!): `uefi_bootmenu` enters the boot menu (with `tianocore_select_bootloader`) to select the device according to BOOTFROM (but doesn't use BOOT_HDD_IMAGE) and start the boot, but after this part `boot_to_desktop` is started and also want to enter the boot menu... So I really don't understand why we have this here, sounds a little bit redundant with `boot_to_desktop`.

And I'm not really sure that this code works, but I can test with it again, and if it still works and in doubt I can let is as is.

Reply:
I'm not sure too, what is the purpose of `boot/uefi_bootmenu`.  Maybe, @asmorodskyi do you remember why we have it here?

Reply:
I just query our codebase - this was only place where this module was called, so probably you can just drop it with this  pool request.

Reply:
So let's drop it (can be restored later, if needed).

Reply:
I wouldn't be so sure about this. Please test openSUSE **and** SLE aarch64 installation tests **and** booting an existing image to be sure about removing it.